samtranslator/schema/aws_serverless_function.py

```diff
@@ -547,6 +555,7 @@ class Globals(BaseModel):
 class Resource(BaseModel):
     Type: Literal["AWS::Serverless::Function"]
     Properties: Optional[Properties]
+    Connectors: Optional[Dict[str, EmbeddedConnector]]
```

Comment:
In a follow-up PR, let's make this part of all resources (currently it's only defined on SAM resources), along with a test that ensures schema validation fails on templates with wrong `Connectors` on non-SAM resource.

We can probably make `Resources` use `allOff` (1) lenient `Connectors` model for any resource, and (2) `anyOf` the current list of resources it validates.

Reply:
Added a TODO